airbyte-integrations/connectors/source-google-ads/source_google_ads/custom_query_stream.py

```diff
@@ -64,6 +64,8 @@ def get_json_schema(self) -> Dict[str, Any]:
             google_data_type = node.data_type.name
             if google_data_type == "ENUM":
                 field_value = {"type": "string", "enum": list(node.enum_values)}
+                if node.is_repeated:
+                    field_value = {"type": ["null", "array"], "items": field_value.copy()}
```

Comment:
If we're concerned about references to `node.enum_values`, we should use `deepcopy(field_value)` instead of `field_value.copy` here.

Reply:
not, really, copy() left from another version of this update